bbn-test-5/README.md

```diff
@@ -8,17 +8,62 @@ in the covenant committee.
 
 ## Network Parameters
 
-<!-- TODO: sync with devops on the things we should add here.
-Off the top of my head:
-* Chain ID
-* Genesis file location
-* Seed nodes
-* RPC nodes
-* Snapshot
-
-We can fully link to our partners here.
+### Babylon Version
+
+<!-- TODO: Version X.Y.Z -->
+
+### Chain ID
+
+`bbn-test-5`
+
+### Genesis
+
+The genesis file can be retrieved from [here](./network-artifacts/genesis.json).
+
+### Seed nodes
+
+<!-- TODO:
+- Nodes.Guru endpoint
+- Polkachu endpoint
+- In-house endpoint (in case we decide to include)
+-->
+
+### Endpoints
+
+#### RPC
```

Comment:
A bit difficult to differentiate between level-3 and level-4 headings. How about we either add numbering to the sections or use a different type of heading for RPC?

Reply:
Added numbering.